src/patternfly/demos/PageLayout/examples/page-layout-expandable-nav-example.hbs

```diff
@@ -1,7 +1,7 @@
 {{#> background-image}}
 {{/background-image}}
 
-{{#> page page--attribute='id="page-layout-expandable-nav"'}}
+{{#> page page--id="page-layout-expandable-nav"}}
```

Comment:
This is the same issue as mentioned above:
`<div class="pf-l-page" page-layout-expandable-nav>`

Reply:
d'oh!